src/lib/hooks/useMapHighlightMarker.ts

```diff
@@ -20,6 +24,7 @@ export function useMapHighlightMarker(
       'w-10 h-10 bg-primary rounded-full ring-2',
       'ring-offset-white ring-offset-2 ring-primary'
     )
+    customMarker.style.backgroundColor = markerViewport.color
```

Comment:
the github warning might be right here, should we add `markerViewport.color` to the effect dependencies?

Reply:
Will do! Thx! 🙏